liberty-ls/src/main/java/io/openliberty/tools/langserver/utils/PropertiesValidationResult.java

```diff
@@ -24,7 +24,7 @@
 import io.openliberty.tools.langserver.model.propertiesfile.PropertiesEntryInstance;
 
 public class PropertiesValidationResult {
-    Integer lineNumber;
+    Integer lineNumber, startChar = null, endChar = null;
```

Comment:
Should `lineNumber` be initialized to null?

Reply:
Another method had set `lineNumber` if the `hasErrors` flag was true. 
I changed the validation entry point to always assign a line number instead.